src/app/ipv/middleware.ts

```diff
@@ -357,6 +357,10 @@ export const handleJourneyPageRequest = async (
     } else if (pageId === PAGES.PYI_TRIAGE_MOBILE_DOWNLOAD_APP) {
       validatePhoneType(context);
       renderOptions.appDownloadUrl = getAppStoreRedirectUrl(context);
+    } else if (pageId === PAGES.PAGE_FACE_TO_FACE_HANDOFF) {
+      renderOptions.postOfficeVisitByDate = new Date().setDate(
+        new Date().getDate() + 15,
```

Comment:
We probably want to pull `15` out into config - it doesn't need to be an env var but we could define it in `config.ts` perhaps?